f90wrap/fortrantype.py

```diff
@@ -78,6 +78,24 @@ def from_handle(cls, handle, alloc=False):
         self._alloc = alloc
         return self
 
+    def __copy__(self):
+        out = FortranDerivedType()
```

Comment:
Does this work correctly for subclasses of `FortranDerivedType`? Doesn't it instead need to be `out = self.__class__()` or similar?

Reply:
It works for subclasses of `FortranDerivedType`, but the output will become `FortranDerivedType` again. The idea is to keep the properties' value but remove the `setter`. If using `self.__class__()`, I couldn't find a way to remove the `setter`. If there is a way to do it, it would be better to use the subclass.